besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1162,6 +1164,7 @@ private void handleUnstableOptions() {
             .put("EVM Options", unstableEvmOptions)
             .put("IPC Options", unstableIpcOptions)
             .put("Chain Data Pruning Options", unstableChainPruningOptions)
+            .put("QBFT options", unstableQbftOptions)
```

Comment:
nit: for consistency

```suggestion
            .put("QBFT Options", unstableQbftOptions)
```